ForkClone.md

```diff
@@ -1,33 +1,33 @@
-# Fork and Clone 
+# Fork and clone 
 
-## Steps to fork the repo: 
+## Steps to fork the repository (repo): 
 
 1. Go to the main GitHub page of the repo:
     (https://github.com/gbowne1/codebooker/)
    
-1. Once you are on the main GitHub page of the repo, click the **Fork** button at the top right of the page. 
+1. Select **Fork** at the top right of the page. 
 
     ![](fork.png)
 
-1. You will now be on a page with the heading **Create a new fork**.
+    This opens a page with the heading **Create a new fork**.
 
-     There are two fields which are required, Owner and Repository name. Make sure the owner is you and the repository name is correct. These fields should already be populated when you clicked to this page in step 2.
+1. Review the prepopulated fields to make sure the owner is you and the repo name is correct.
 
     ![](CreateFork.png)
 
-2. Click on the green **Create fork** button on the lower right hand corner of the page.
+1. Select **Create fork** in the lower right corner of the page.
```

Comment:
I shortened "right hand" to "right" for brevity. If we'd have kept it, we should probably hyphenate it because many style guides say to hyphenate it when it's a compound modifier.